Mathlib/Algebra/Module/Submodule/Defs.lean

```diff
@@ -212,6 +212,12 @@ lemma smul_mem_iff'' [Invertible r] :
   rw [← invOf_smul_smul r x]
   exact p.smul_mem _ h
 
+@[simp]
+lemma smul_mem_iff_IsUnit (hr : IsUnit r) :
```

Comment:
```suggestion
lemma smul_mem_iff_of_isUnit (hr : IsUnit r) :
```
And I am really sad that we are adding a lemma to a `Def.lean` file but I guess this is not your problem.